packages/forms/stories/ErrorsForm.stories.tsx

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable import/no-unresolved */
+import React, { useEffect } from 'react';
+import { Provider, useDispatch, useSelector } from 'react-redux';
+
+import { storiesOf } from '@storybook/react';
+
+import Input, { IInputProps } from '@xcritical/input';
+
+import Form, {
+  formSelector,
+  xcriticalFormError,
+  xcriticalFormInit,
+  xcriticalFormShowErrors,
+  xcriticalFormSetFieldsMeta, FormField,
+} from '../src';
+
+import { InputWithErrorWrapper } from './SampleComponents';
+import store from './store';
+
+
+const errors = {
+  firstName: 'error to firstName field',
+  lastName: 'error to lastName field',
+  age: 'error to age field',
+};
+
+const initData = {
+  firstName: '123',
+  lastName: '123',
+  age: '123',
+};
+
+const FORM_NAME = 'error-form';
+
+const validate = (model) => ({
+  firstName: model.firstName === '123' ? errors.firstName : null,
+  lastName: model.lastName === '123' ? errors.lastName : null,
+  age: model.age === '123' ? errors.age : null,
+});
+
+const InputWithError = ({ error, ...props }: IInputProps & { error?: string}) => (
+  <InputWithErrorWrapper>
+    <Input { ...props } />
+    <br />
+    { error && <span>{ error }</span> }
+  </InputWithErrorWrapper>
+);
+
+const changedMeta = {
+  firstName: {
+    touch: true,
+    changed: true,
+  },
+  lastName: {
+    touch: true,
+    changed: true,
+  },
+  age: {
+    touch: false,
+    changed: false,
+  },
+};
+
+
+storiesOf('Form', module)
+  .addDecorator((Story) => (
+    <Provider store={ store }>
+      <Story />
+    </Provider>
+  ))
+  .add('Error Form', () => {
+    const dispatch = useDispatch();
+    const form = useSelector((state) => formSelector(state, FORM_NAME));
+
+    useEffect(() => {
+      const validationErrors = validate(form.model);
+      dispatch(xcriticalFormError(FORM_NAME, validationErrors));
+    }, [form.model]);
```

Comment:
Dispatch is stable function, hovewer, linter can show warnings about missing dispatch in useEffect dependency. 
In react-redux documentation authors advise to add dispatch as a dependency.